rector.php

```diff
@@ -18,14 +17,17 @@
         __DIR__ . '/src/*/src',
     ]);
 
+    $parameters->set(Option::PARALLEL, true);
     $parameters->set(Option::SKIP, [
         CountOnNullRector::class,
 
         // for PHP 8
         RemoveUnusedPromotedPropertyRector::class,
 
-        // buggy when remove private property, but filled by construct in the middle of parameter
-        RemoveUnusedPrivatePropertyRector::class,
+        RemoveUnusedPrivatePropertyRector::class => [
+            // buggy when used in middle of parameter along with RemoveUnusedConstructorParamRector that may cause BC break
+            __DIR__ . '/src/Console/src/Sequence/CallableSequence.php',
```

Comment:
this skipped part resolved at https://github.com/rectorphp/rector-src/pull/1714 , can be cleaned up on next rector release.